modules/migration-configuring-gcp.adoc

```diff
@@ -19,7 +20,12 @@ You can configure a Google Cloud Provider (GCP) storage bucket as a replication
 
 .Procedure
 
-. Run `gsutil init` to log in:
+. Log in to `gsutil`:
```

Comment:
Question: Is it a log in to `gsutil` or to Google Cloud? It might be the distinction doesn't matter though.

Reply:
It's logging in to gsutil